### ISSUE TITLE
First approach to section 6 (Runtime view)

### ISSUE THREAD

**Issue by UO283216** [Arquisoft/lomap_en2a#24]

**Comment by UO283216** [comment]
I am currently creating this section and deciding some interesting ideas that we should apply to the design of the application. I think it comes really handy to use the structure we used in one of the DS lab assigments where we implemented a map. We use the Adapter pathern to decouple the  implementation of the map with the application map, this way our application will have some sort of façade also, it will only communicate with what I've named "MapAdapter" and this object is the one in charge of receiving the requests of our application and adapting them to the map API calls, it will responde to the domain needs with a perfect method that for example "addPlaceMark(long, lat, title, image)" that will perfom the addition of the point to the actual map, the renaming and the assignation of the image for the mark created.
